pipelines/README.md

```diff
@@ -91,15 +91,23 @@ oc describe imagestream/tensorflow-housing
 
 ### Quay Repository and Robot Permissions
 
-- In your Quay namespace, create repository `tensorflow-housing`, add a robot account to push images and set write Permissions for the robot on the repository.
-- Download `build-secret.yml`
-- Apply build-secret. E.g.:
+- In your Quay namespace:
+  - Create repositories `tensorflow-housing` and/or `bike-rentals-auto-ml`.
+  - Add a robot account to push images and set write Permissions for the robot account on the repositories.
+  - Download the Kubernetes Secret of the robot account and store it in a YAML file.
```

Comment:
Quay already lets you download the secret in `YAML` format?

![image](https://github.com/opendatahub-io/ai-edge/assets/99439005/41bc0a22-a8ac-40d4-903e-b482837484d5)

Reply:
Yes, it's that "Download rhoai-edge-osp-secret.yml" link in that screenshot.